app/src/main/res/layout/nav_header.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="@dimen/navdrawer_width"
```

Comment:
Since you delete this [line](https://github.com/gdg-x/frisbee/pull/379/files#diff-45271120bee999ce89e5e0a189e4451dL22) below in `navdrawer.xml` you should set the width here to `match_parent` I believe. 
This makes the header image slightly smaller then the NavDrawer itself. You can see the white space on your screenshot. 

Thank you. :smile: